msg/en_US/utils.msg

```diff
@@ -1330,5 +1330,5 @@ valid options:\n\
                                   WARNING: This option has to be used with '-t(--transaction)' option.\n\
     -m, --module=TYPE             display detailed memory usage information of the specified TYPE.\n\
                                   TYPE of modules\n\
-                                  0 (or all)      all modules\n\
-                                  1 (or heap)     heap module\n
+                                  -1      all modules\n\
+                                   0      heap module\n
```

Comment:
```suggestion
```
how about adding heap module when memory monitoring for heap module is implemented? 

Since developers can see which files were modified when the heap module was added, developers looking to implement memory monitoring for other modules might find it easy to refer to

Reply:
You're right. I remove it.